src/js/providers/html5.ts

```diff
@@ -179,11 +179,18 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         },
 
         canplay(this: ProviderWithMixins): void {
-            _canSeek = true;
-            if (!_androidHls) {
-                _setMediaType();
+            if (OS.iOS && Browser.safari) {
+                return;
             }
-            checkVisualQuality();
+            _canPlayHandler();
+            VideoEvents.canplay.call(_this);
+        },
+
+        canplaythrough(this: ProviderWithMixins): void {
+            if (!(OS.iOS && Browser.safari)) {
```

Comment:
The browser doesn't matter. All browsers in iOS operate the same. Remove `Browser.safari` check.